data/src/main/kotlin/com/antosik/benchproject/data/common/modules/movies/MoviesModule.kt

```diff
@@ -27,4 +29,8 @@ internal object MoviesModule {
     @Provides
     fun provideMovieDetailsRepository(apiService: MoviesApi, database: MovieDatabase): MovieDetailsRepository =
         MovieDetailsDataRepository(apiService, database.dao)
+
+    @Provides
+    fun provideFavouriteMoviesRepository(database: MovieDatabase): FavouriteMoviesRepository =
```

Comment:
Let's decide and use one version of `favorite` word :D i see that in few places different versions are used :D